multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -36,14 +36,21 @@ coreos:
       command: start
       enable: true
       content: |
+        [Unit]
+        Requires=load-rkt-stage1.service
+        After=load-rkt-stage1.service
         [Service]
+        ExecStartPre=/usr/bin/touch /etc/hosts
```

Comment:
I thought this was being populated now: https://github.com/coreos/baselayout/pull/50